src/extension/atom.rs

```diff
@@ -0,0 +1,100 @@
+// This file is part of rss.
+//
+// Copyright © 2015-2020 The rust-syndication Developers
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the MIT License and/or Apache 2.0 License.
+
+use std::collections::HashMap;
+use std::io::Write;
+
+use atom_syndication::Link;
+use quick_xml::events::{BytesStart, Event};
+use quick_xml::Error as XmlError;
+use quick_xml::Writer;
+
+use crate::extension::util::remove_extension_value;
+use crate::extension::Extension;
+use crate::toxml::ToXml;
+
+/// The Atom XML namespace.
+pub const NAMESPACE: &str = "http://www.w3.org/2005/Atom";
+
+/// An Atom element extension.
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Default, Debug, Clone, PartialEq)]
+#[cfg_attr(feature = "builders", derive(Builder))]
+#[cfg_attr(feature = "builders", builder(setter(into), default))]
+pub struct AtomExtension {
+    /// Link
+    pub link: Option<Link>,
```

Comment:
I could be wrong, but I don't think it's correct to state "you can only have one link tag"

Reply:
You are right. There can be many links.